misc/python/materialize/docker.py

```diff
@@ -0,0 +1,64 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+"""Docker utilities."""
```

Comment:
Not sure if docker.py is the perfect file for this, I expected version from docker at first, not version from git.

Reply:
I am not returning a git version but rather an image tag.

Reply:
I slightly improved that with https://github.com/MaterializeInc/materialize/pull/23145/commits/5016040098628696e0fcb2c8b36ac53e691f45cf.